terraform/stacks/external/outputs.tf

```diff
@@ -3,10 +3,10 @@ output "cdn_broker_username" {
   value = "${module.cdn_broker.username}"
 }
 output "cdn_broker_access_key_id_prev" {
-  value = "${module.cdn_broker.access_key_id_prev}"
+  value = ""
```

Comment:
so does the same thing revert need to happen for `terraform/modules/cloudfoundry/outputs.tf` and `terraform/stacks/main/outputs.tf` as well? @jmcarp